### PR TITLE
Axes

### PR DESCRIPTION
**Pull request by jchavesmontero**
I added the option to extract the power spectrum from different axes

### CODE REVIEW

File: lace_fake/snapshot_admin.py [hunk 1]
```diff
@@ -24,14 +24,21 @@ def __init__(self,snap_json,scales_tau=None,kF_Mpc=None,post_dir=None):
             print('use',post_dir,'as post-processing directory')
             self.data['post_dir']=post_dir
             self.data['raw_dir']=None
-            self.data['skewers_dir']=post_dir+'/skewers/'
         else:
             if 'post_dir' in self.data:
                 print('no post_dir provided, will use',self.data['post_dir'])
-                self.data['skewers_dir']=self.data['post_dir']+'/skewers/'
+                # self.data['skewers_dir']=self.data['post_dir']+'/skewers/'
```
Comment (andreufont) [inline lace_fake/snapshot_admin.py:30]: We should think of all possible use cases here. Old files from Chris (that we want to use for the P3D emulator) do not have post_dir in the json files. However, I guess for those we should pass post_dir explicitly, so it is probably fine. I guess we can merge and come back to fix this if something is crashing.

File: lace_fake/snapshot_admin.py [hunk 2]
```diff
@@ -24,14 +24,21 @@ def __init__(self,snap_json,scales_tau=None,kF_Mpc=None,post_dir=None):
...
             else:
                 raise ValueError('must provide post_dir when using old files')
 
+        self.data['axis'] = axis
```
Comment (andreufont) [inline lace_fake/snapshot_admin.py]: I think we can delete this line, since we are assigned it again in the if/else code block.

Comment (jchavesmontero) [reply lace_fake/snapshot_admin.py]: Yes, I was doing something different before, good catch!

File: lace_fake/snapshot_admin.py (hunk 1 above)
Comment (jchavesmontero) [reply lace_fake/snapshot_admin.py:30]: Note that the original function was:

```
if post_dir:
            print('use',post_dir,'as post-processing directory')
            self.data['post_dir']=post_dir
            self.data['raw_dir']=None
            self.data['skewers_dir']=post_dir+'/skewers/'
        else:
            if 'post_dir' in self.data:
                print('no post_dir provided, will use',self.data['post_dir'])
                self.data['skewers_dir']=self.data['post_dir']+'/skewers/'
            else:
                raise ValueError('must provide post_dir when using old files')
```

so I did not change anything there. Anyway, let's figure out if crashes

**Comment by jchavesmontero** [comment]
Should I merge now?

**Review by andreufont — APPROVED** [review]
I think the changes look good, but I'd like to think a bit more about all possible corner cases (using old files without post_dir, and so on). Let's chat tomorrow in person.

File: lace_fake/snapshot_admin.py (hunk 1 above)
Comment (andreufont) [reply lace_fake/snapshot_admin.py:30]: You are right, you are defining skewers_dir later on now. I wrote my comment too early.

**Comment by andreufont** [comment]
I forgot that you were defining skewers_dir in the same way we were doing before, you just do it later on. I'd be happy to merge now.

**Pull request merged**